gc/base/MarkingScheme.cpp

```diff
@@ -435,3 +435,29 @@ MM_MarkingScheme::setupIndexableScanner(MM_EnvironmentBase *env, omrobjectptr_t
 	return _delegate.setupIndexableScanner(env, objectPtr, reason, sizeToDo, sizeInElementsToDo, basePtr, flags);
 }
 
+void
+MM_MarkingScheme::markObjectsForRange(MM_EnvironmentBase *env, void *heapBase, void *heapTop)
```

Comment:
as a follow-up, try to have SegregateMS call this method (adjusts top address before it)

Reply:
Done as part of these changes